### ISSUE TITLE
Problem with installation on ubuntu 19.10

### ISSUE THREAD

**Issue by chriad** [Illumina/Isaac4#13]
I get the following error when trying to install:

```
[ 49%] Building CXX object c++/lib/alignment/CMakeFiles/isaac_alignment.dir/TemplateBuilder.cpp.o
/home/chriad/Desktop/nebula/isaac/Isaac4/src/c++/lib/alignment/TemplateBuilder.cpp: In constructor ‘isaac::alignment::TemplateBuilder::TemplateBuilder(bool, const FlowcellLayoutList&, std::size_t, unsigned int, unsigned int, unsigned int, unsigned int, unsigned int, unsigned int, bool, bool, bool, bool, unsigned int, unsigned int, bool, unsigned int, bool, const isaac::alignment::AlignmentCfg&, isaac::alignment::TemplateBuilder::DodgyAlignmentScore, unsigned int, bool)’:
/home/chriad/Desktop/nebula/isaac/Isaac4/src/c++/lib/alignment/TemplateBuilder.cpp:74:30: error: ‘pow10’ was not declared in this scope
   74 |     , anomalousPairHandicap_(pow10(double(anomalousPairHandicap) / 10.0) - 1.0)//anomalousPairHandicap)
      |                              ^~~~~
make[2]: *** [c++/lib/alignment/CMakeFiles/isaac_alignment.dir/build.make:206: c++/lib/alignment/CMakeFiles/isaac_alignment.dir/TemplateBuilder.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:998: c++/lib/alignment/CMakeFiles/isaac_alignment.dir/all] Error 2

```

Does it have to do with the version of glibc? (see https://abi-laboratory.pro/?view=changelog&l=glibc&v=2.27)

> * The libm functions pow10, pow10f and pow10l are no longer supported for
>   new programs.  Programs should use the standard names exp10, exp10f and
>   exp10l for these functions instead.

I use version 2.30

```
 $ /lib/x86_64-linux-gnu/libc.so.6 --version
GNU C Library (Ubuntu GLIBC 2.30-0ubuntu2.1) stable release version 2.30.
```

What can I do to get it running?

**Comment by rpetrovski** [comment]
Hello, I've pushed some fixes for ubuntu 20.04. Let me know if you still have problems.

R.

**Comment by chriad** [comment]
It works, thank you!
